pkg/config/profile.go

```diff
@@ -26,6 +27,25 @@ func (p *Profile) CreateProfile() error {
 	return nil
 }
 
+func (p *Profile) GetColor() (string, error) {
+	color := viper.GetString("color")
```

Comment:
Just making sure, this reads properly from the command-line argument?

Reply:
Yep! This correctly grabs the command-line argument and overrides the stored config.